amethyst_ui/src/event.rs

```diff
@@ -48,6 +48,10 @@ pub enum UiEventType {
         /// The entity on which the dragged object was dropped.
         dropped_on: Entity,
     },
+    /// When the value of an `Editable` Ui element has changed.
+    Change,
+    /// When an `Editable` Ui element has been committed.
```

Comment:
There's no Editable component. Should this documentation be changed to say Selectable?

Reply:
@jojolepro Good point. Would `TextEditing` be a more appropriate component reference? Perhaps it's more clear to the end user to just use a phrase like: 

> When the value of an editable `UiText` element has been changed by user input.

Reply:
I think it should at least be called `ValueChange`. I like your proposed description, it clarifies that the change is due to user input, which I think is the important part.

Reply:
Value change would not indicate the type of the value changed. ie checkbox vs text

Imo TextEdited would be a good name.

Reply:
:+1: